src/scandeval/benchmarker.py

```diff
@@ -130,6 +132,7 @@ def __init__(
         verbose: bool = False,
         trust_remote_code: bool = False,
         load_in_4bit: bool | None = None,
+        use_flash_attention: bool = False,
```

Comment:
Any reason why this is not `True`

Reply:
I assumed as much. We might accidentally bias the benchmark by requiring flash attention for some models, but I if the package is problematic to build I would keep it as is. Pytorch has also integrated flash attention v1 so I would imagine that v2 will also become native at some point.